Formula/c/cfv.rb

```diff
@@ -16,7 +18,6 @@ class Cfv < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "398b65a6b3495ccc1cff405108a5db678ca99b6b7c3d355fcf450c8ec2ea3ea9"
   end
 
-  depends_on "python-setuptools" => :build
   depends_on "python@3.12"
 
   def python3
```

Comment:
@ZhongRuoyu Is `def python3` still required?

Reply:
I realised there's no need for `virtualenv_install_with_resources` and we can just remove `python-setuptools` straight away. Updating the rest of PRs too.